sdk/core/core-util/test/public/createAbortablePromise.spec.ts

```diff
@@ -5,7 +5,7 @@ import * as sinon from "sinon";
 import { AbortController, AbortSignalLike } from "@azure/abort-controller";
 import chai from "chai";
 import chaiAsPromised from "chai-as-promised";
-import { createAbortablePromise } from "../../src/createAbortablePromise";
+import { createAbortablePromise } from "../../src/aborterUtils";
```

Comment:
can we add tests for the new abstraction?

Reply:
Added some